userspace/libsinsp/threadinfo.cpp

```diff
@@ -1444,14 +1444,6 @@ void sinsp_thread_manager::clear()
 	m_last_tid = 0;
 	m_last_flush_time_ns = 0;
 	m_n_drops = 0;
-
```

Comment:
SInce we have removed all these fields at this point i would remove also `m_n_drops` and add it to sinsp_stats. Maybe we could use a more meaningful name like `full_threadtable_drops`. In this way we can handle this stat like all the others without a dedicated method `get_m_n_drops`

Reply:
Done, but came at a small costs in terms of changing a g_logger log that now only fires if sinsp stats v2 are enabled.